app/models/post.rb

```diff
@@ -2,4 +2,9 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   has_many :comments
   has_many :likes
+
+  # TO Use : p = Post.find(1); p.update_posts_counter
+  def update_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
```

Comment:
### [OPTIONAL]
- [ ] It will be great to use [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the` update_post_counter `method private